index.html

```diff
@@ -1,14 +1,48 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <nav>
+            <a href="">
+                <h4>My Logo</h4>
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
```

Comment:
- [ ] kindly you should not wrap the logo and hamburger button in the nav tag to follow HTML best practices. But the hamburger button should have a `nav` tag to contain nav links.you got what it takes 👍